pkg/expression/integration_test/integration_test.go

```diff
@@ -2771,72 +2771,6 @@ func TestIssue16205(t *testing.T) {
 	require.NotEqual(t, rows1[0][0].(string), rows2[0][0].(string))
 }
 
-// issues 14448, 19383, 17734
-func TestNoopFunctions(t *testing.T) {
```

Comment:
FT has no cache and it is so slow. Is it unnecessary to move?

Reply:
@hawkingrei I think we can discuss this topic in #45961 with @Defined2014.

Reply:
Both are fine for me. If it uses ft, it can run it in both `tikv` and `tidb` environments, but it will be slower because there is no cache.